botCommands/note.js

```diff
@@ -0,0 +1,12 @@
+const { registerBotCommand } = require("../botEngine.js");
+
+const command = {
+  regex: /(?<![\w`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~])\/note(?![\w`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~)])/,
```

Comment:
This regex is overly complicated. Take a look at a different example to get a better idea on how to do this. This is unreadable to a normal human and could take a LONG time to understand if there's a bug within.

Reply:
I also believe we should call it "notes" instead of "note". Nobody just makes one note.

Reply:
https://github.com/TheOdinProject/odin-bot-v2/blob/master/botCommands/callbacks.js

See here for example.

It's often very useful to look at the patterns established by other pieces of code before trying to establish your own.

Reply:
> This regex is overly complicated. Take a look at a different example to get a better idea on how to do this. This is unreadable to a normal human and could take a LONG time to understand if there's a bug within.
Yes, I agree it is overly complicated :( I was having a hard time to pass all the tests that were provided and that one above was the one I came with that could pass all the tests. 

> I also believe we should call it "notes" instead of "note". Nobody just makes one note.
I will change it to the 'notes' instead of 'note'.

> It's often very useful to look at the patterns established by other pieces of code before trying to establish your own.
Yes, I initially went with this one actually, but it couldn't pass the tests if it's followed by any non word characters like '/', '.', '*', and etc. Any ideas how I can do it??? Or should I just leave it for the time being?

For example, `/\B\/notes` will not pass '/notes!', because it's followed by '!'.